docs/source/upgrade_notes.rst

```diff
@@ -28,8 +28,40 @@ Upgrade Notes
    /opt/stackstorm/st2/bin/pip install "git+https://github.com/StackStorm/stackstorm-runner-windows.git#egg=stackstorm-runner-windows"
 
    sudo st2ctl reload --register-runners
- 
- * The :doc:`Inquiries </inquiries>` API has been promoted from the ``/api/exp`` path to ``/api/v1``. If you have any external systems that use this API they will need to be updated to use the new path. st2client has been updated to use the new path.
+* The :doc:`Inquiries </inquiries>` API has been promoted from the ``/api/exp`` path to ``/api/v1``.
+  If you have any external systems that use this API they will need to be updated to use the new
+  path. st2client has been updated to use the new path.
+* If you are using |EWC| with RBAC you need to update your ``/etc/st2/st2.conf`` config file for RBAC
+  to work after the upgrade.
```

Comment:
I thought this migration should be done by the corresponding package we add & install as a dependency?

Reply:
It does happen automatically now inside post install, yes (everything except restarting the services).

I left it here just in case (to ensure option is indeed there) and also to serve as a "reference" when this option was added.